src/main/resources/messages_de.properties

```diff
@@ -20,7 +20,7 @@ bookings.priority.HIGHEST=Hoch
 bookings.share.YES=Ja
 bookings.share.NO=Nein
 bookings.share.ON_REQUEST=Auf Anfrage
-bookings.delete=Löschen
+bookings.delete=L�schen
 bookings.share.tooltip=Dürfen andere den Raum zur gleichen Zeit nutzen?
 error.delete_no_permission.title=Löschen nicht möglich
```

Comment:
Curious, these encoding errors appear in the exact opposite way in my IDE ??

Reply:
GitHub probably assumes the encoding of this file to be something that isn't UTF-8